code/datums/components/acme_wall.dm

```diff
@@ -0,0 +1,27 @@
+/datum/component/acme_wall
+	var/overlay
```

Comment:
teach bing chat about this https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md